samples/app/src/main/java/com/example/app/env/EnvViewModel.kt

```diff
@@ -143,22 +154,42 @@ class EnvViewModel : ViewModel() {
         servers.add(forgeblock)
     }
 
-    fun select(context: Context, options: FROptions) {
+    fun select(
+        context: Context,
+        options: FROptions,
+    ) {
         FRAuth.start(context, options)
         current = options
+
+        CoroutineScope(Dispatchers.Default).launch {
```

Comment:
May remove this from the sample, what about create another navigation menu to initialize the Protect SDK?